<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result</class>
 <widget class="QMainWindow" name="Result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Результат</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Система</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="systemTW">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>115</number>
      </attribute>
      <column>
       <property name="text">
        <string>Всего</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>% обработанных</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>% ушедших</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Очереди</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="queueTW">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>930</width>
        <height>100</height>
       </size>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>макс. фишек</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>попытки входа</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>пустых входов</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>средний размер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>среднее время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>кол-во покинувших</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Группы окон</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="windowsTW">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <column>
       <property name="text">
        <string>название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>обработано</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>загрузка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ср. время работы</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
